learn-resources/data/notification-web.json

```diff
@@ -0,0 +1,8 @@
+{
+	"general": {
+		"en_US": {
+			"message": "Learn more.",
```

Comment:
Hey @larissacribeiro, could you check if this is the right message?

Reply:
yes, @igor-franca, I've left a comment above yours explaining this change!